contrib/cirrus/packer/fedora_packaging.sh

```diff
@@ -37,7 +37,16 @@ fi
 
 $BIGTO ooe.sh $SUDO dnf update -y
 
+# While both runc and crun are needed in the support of different distros
+# and environments, packaging will need to instal one and suggesting the other.
+# This can cause some ambiguity for automation, where more concrete, predictable
+# behavior is desirable.  Instead of fighting with or complicating the packaging
+# itself, workaround the problem.  Always removing 'runc' in Fedora since it's
+# exclusively cgroupsv2 focused as of release 31.  However, ensure the runc
+# is downloaded to $PACKAGE_DOWNLOAD_DIR for the minority of use-cases where
+# automation needs to swap it in.
```

Comment:
I'm having a lot of trouble parsing this. Might I suggest something like:
```
# Fedora, as of 31, uses cgroups v2 by default. runc does not support 
# cgroups v2, only crun does. (As of 2020-07-30 runc support is
# forthcoming but not even close to ready yet). To ensure a reliable
# runtime environment, force-remove runc if it is present.
# (Something something something automation, whatever that means)
```

Reply:
good suggestions.

FWIW: "something something" == other repos need runc (and other packages) so we download and cache them at build-time so they can be installed at runtime w/o version/update surprises :grin:

Reply:
okay, got it.